stdlib/rational.gr

```diff
@@ -19,17 +19,53 @@ from Numbers use {
   timesDivideRational,
   rationalsEqual,
   cmpRationals,
+  isInteger,
 }
 
 provide { fromNumber, toNumber, numerator, denominator }
 
+/**
+ * Gets the numerator and denominator of the rational as a tuple.
+ *
+ * @param x: The rational to split
+ * @returns The numerator and denominator of the rational
+ *
+ * @since v0.6.0
+ */
+provide let toIntegerRatio = x => {
+  (numerator(x), denominator(x))
+}
+
+/**
+ * Creates a rational from a numerator and denominator.
+ *
+ * @param numerator: The numerator
+ * @param denominator: The denominator
+ * @returns The newly constructed rational
+ *
+ * @throws InvalidArgument(String): If the numerator or denominator are not integers
```

Comment:
Split this into two throws "if the numerator is not an integer" and "if the denominator is not an integer"